CONTRIBUTING.rst

```diff
@@ -36,34 +36,41 @@ can be contributed to scikit-image.
      <https://github.com/scikit-image/scikit-image>`_ and click the
      "fork" button to create your own copy of the project.
 
+   * `Set up GitHub SSH authentication <https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh>`__.
+
    * Clone (download) the repository with the project source on your local computer::
 
-      git clone https://github.com/your-username/scikit-image.git
+      git clone --origin upstream git@github.com:scikit-image/scikit-image
 
    * Change into the root directory of the cloned repository::
 
       cd scikit-image
 
-   * Add the upstream repository::
+   * Add your fork as a
+     `remote repository <https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes>`__
+     that you will interact with.
+
+     Assuming a GitHub username of `codemonkey`::
 
-      git remote add upstream https://github.com/scikit-image/scikit-image.git
+        git remote add codemonkey git@github.com:codemonkey/scikit-image
+        git fetch codemonkey
 
-   * Now, you have remote repositories named:
+   * You now have two remote repositories:
 
-     - ``upstream``, which refers to the ``scikit-image`` repository, and
-     - ``origin``, which refers to your personal fork.
+     - ``upstream``, which refers to the ``scikit-image`` project repository, and
+     - ``codemonkey``, which refers to your personal fork.
 
    * Next, :ref:`set up your build environment <build-env-setup>`.
 
-   * Finally, we recommend that you use a pre-commit hook, which runs code
+   * Finally, we recommend that you use our pre-commit hook, which runs code
      checkers and formatters each time you do a ``git commit``::
 
        pip install pre-commit
        pre-commit install
 
 2. Develop your contribution:
 
-   * Pull the latest changes from upstream::
+   * Pull the latest changes from the project::
 
       git switch main
       git pull upstream main
```

Comment:
Again, I do it this way, but it's the old way... Git now suggests we should fetch first, then merge (iirc).